pkg/controller/risingwave_controller.go

```diff
@@ -336,6 +340,35 @@ func (c *RisingWaveController) reactiveWorkflow(risingwaveManger *object.RisingW
 	)
 	sharedSyncAllAndWait := ctrlkit.Shared(syncAllAndWait)
 
+	releaseScaleViewLock := mgr.NewAction(RisingWaveAction_ReleaseScaleViewLock, func(ctx context.Context, l logr.Logger) (ctrl.Result, error) {
+
+		risingWave := risingwaveManger.RisingWaveReader.RisingWave()
+		scaleViews := risingWave.Status.ScaleViews
+		lockIdx := make([]int, 0)
+
+		for i, s := range scaleViews {
+			var scaleView risingwavev1alpha1.RisingWaveScaleView
+			err := c.Client.Get(ctx, types.NamespacedName{
+				Namespace: risingWave.Namespace,
+				Name:      s.Name,
+			}, &scaleView)
+
+			if err != nil {
+				if apierrors.IsNotFound(err) {
+					l.V(1).Info("Not found, abort")
+					lockIdx = append(lockIdx, i)
+				} else {
+					l.Error(err, "Failed to get risingwavescaleview")
```

Comment:
Shall we break immediately and return an error here? Since it seems the error is unexpected and fails to handle.